src/services/stringCompletions.ts

```diff
@@ -114,6 +114,10 @@ namespace ts.Completions.StringCompletions {
                 switch (parent.parent.kind) {
                     case SyntaxKind.TypeReference:
                         return { kind: StringLiteralCompletionKind.Types, types: getStringLiteralTypes(typeChecker.getTypeArgumentConstraint(parent as LiteralTypeNode)), isNewIdentifier: false };
+                    case SyntaxKind.ParenthesizedType: {
+                        const typeNode = findAncestor(parent.parent, n => !isParenthesizedTypeNode(n.parent)) as ParenthesizedTypeNode;
+                        return { kind: StringLiteralCompletionKind.Types, types: getStringLiteralTypes(typeChecker.getTypeArgumentConstraint(typeNode)), isNewIdentifier: false };

```

Comment:
This doesn't seem right - you don't necessarily know whether you're in a "type argument constraint" position. That's the issue. For example, you could have

```ts
type Foo = Bar[  (  (  "/**/"  )  )  ];
```

in which case, you actually want to give results from the `IndexedAccessType` case, right?

Reply:
The fix handles the case `<(( "/**/" ))>` which was described in the bug description, it doesn't affect completion for `IndexedAccessType`.